server/services/GameLogic.js

```diff
@@ -117,7 +117,9 @@ class GameLogic {
             this.setStatus(statusTypes.GAME_OVER);
             socket.emitStatus(this.roomId, this.status);
             const winner = this.players.reduce((prev, current) => (prev.score > current.score) ? prev : current);
+            const drawers = this.players.reduce((prev, current) => (prev.score === current.score) ? [prev, current] : []);
             socket.emitWinner(this.roomId, { username: winner.username });
+            socket.emitDrawers(this.roomId, drawers);
```

Comment:
Currently this is going to emit winner and drawers, can you make it so that it emits just one depending on whether there is a draw or not?